developers_chamber/ecs_utils.py

```diff
@@ -463,39 +464,48 @@ def run_task_and_wait_for_success(cluster, task_definition, command, name, succe
     task_id = task.split('/')[-1]
 
     LOGGER.info("Running task: '{}'".format(task))
-
     wait_for_task_to_stop(cluster=cluster, task=task, timeout=timeout, region=region, ecs_client=ecs_client)
-
     response = ecs_client.describe_tasks(cluster=cluster, tasks=[task])
 
     UNDEFINED = 'UNDEFINED'
 
     task_stop_code = response['tasks'][0].get('stopCode', UNDEFINED)
     LOGGER.info("Task stop code: '{}'".format(task_stop_code))
 
+    container_response = {}
+    for _container_response in response['tasks'][0]['containers']:
+        if _container_response['name'] == name:
+            container_response = _container_response
+
     if task_stop_code != 'EssentialContainerExited':
-        contianer_stop_reason = response['tasks'][0]['containers'][0].get('reason', UNDEFINED)
+        contianer_stop_reason = container_response.get('reason', UNDEFINED)
         LOGGER.info("Container stop reason: '{}'".format(contianer_stop_reason))
 
         if contianer_stop_reason != UNDEFINED:
             raise ClickException(contianer_stop_reason)
 
         task_stop_reason = response['tasks'][0].get('stoppedReason', UNDEFINED)
         LOGGER.info("Task stop reason: '{}'".format(task_stop_reason))
-
         if task_stop_reason != UNDEFINED:
             raise ClickException(task_stop_reason)
 
         raise ClickException(response['tasks'][0])
 
     try:
+        LOGGER.info('Task output:')
```

Comment:
tady jsem si zlepšil výstup, odsadil jsem ho + jsem tam přidal datum a čas události, pomůže nám to při testování migrací, budeme vědět jak dlouho běžely